Branch-SDK/src/io/branch/referral/RemoteInterface.java

```diff
@@ -344,6 +343,9 @@ private ServerResponse make_restful_post(JSONObject body, String url, String tag
             long reqStartTime = System.currentTimeMillis();
             OutputStreamWriter outputStreamWriter = new OutputStreamWriter(connection.getOutputStream());
             lastRoundTripTime_ = (int) (System.currentTimeMillis() - reqStartTime);
+            if (Branch.getInstance() != null) {
+                Branch.getInstance().addExtraInstrumentationData(tag + "-" + Defines.Jsonkey.Last_Round_Trip_Time.getKey(), "" + lastRoundTripTime_);
```

Comment:
Do you think we can use `Integer.toString(lastRoundTripTime_)` here? I'd rather avoid `"" + int`